src/anupSJun20/PhoneNumberValidation.java

```diff
@@ -0,0 +1,76 @@
+package anupSJun20;
+/*Assignment 24 :- 5th Jul'2020 
+Validate given phone number.
++91 9765463742 --> valid
+09765463742 --> valid
+976 546 3742 --> valid
+9765463742 --> valid
+9765463A42 --> Invalid
+976546342 --> Invalid
+6976546342 --> Invalid
+Note : phone number is given in String format.*/
+
+public class PhoneNumberValidation {
+
+	static void findPhoneNumberhavingCharacter(String number) {
+		boolean flag = true;
+		for (int index = 0; index < number.length(); index++) {
+			if (Character.isLetter(number.charAt(index))) {
+				System.out.println("The phone number : " + number + " is invalid ");
+				flag = false;
+			}
+		}
+		if (flag) {
+			verifyPhoneNumber(number);
+		}
+	}
+
+	static void verifyPhoneNumber(String number) {
+
+		if (number.length() < 10 || number.length() > 14) {
+			System.out.println("The phone number : " + number + " is invalid ");
+		} else if (number.length() == 14) {
+			if (number.startsWith("+91") && number.charAt(3) == ' ' || number.charAt(4) == '7'
+					|| number.charAt(4) == '8' || number.charAt(4) == '9') {
```

Comment:
For space you should check contsins(" ") and for numbers checks number.startsWith("7/8/9")

Reply:
same for other conditions

Reply:
Updated as per the comment. Please review.